gef.py

```diff
@@ -6404,6 +6404,46 @@ def do_invoke(self, argv):
         gef_print(chunk.psprint())
         return
 
+@register_command
+class GlibcHeapChunkRangeCommand(GenericCommand):
+    """Display information on all heap chunks within memory region {from, to}.
+    First address should be valid chunk address while the last one could be arbitrary > first.
```

Comment:
* {from, to} => [from, to)
* should be a valid chunk address
* could be arbitrary, as long as it is higher than the first